.github/workflows/release.yml

```diff
@@ -11,36 +11,25 @@ on:
         default: 'false'
         required: false
 
+env:
+  REGISTRY_NAME: registry.scality.com
+  PROJECT_NAME: ${{ github.event.repository.name }}
+
 jobs:
-  release:
+  build:
+    name: Build and Push
+    uses: ./.github/workflows/build.yml
+    with:
+      namespace: s3utils
+      tag: ${{ github.event.inputs.tag }}
+    secrets: inherit
+
+  github-release:
+    name: Create GitHub Release
     runs-on: ubuntu-latest
+    needs:
+      - build
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
```

Comment:
Just curious, why did we remove this step in `.github/workflows/build.yml` ?

Reply:
oh, it's a mistake! I should copy it to `build.yml` file
thanks a lot Walid!